file/provider/delete_volume_access_point.go

```diff
@@ -70,7 +69,7 @@ func (vpcs *VPCSession) DeleteVolumeAccessPoint(deleteAccessPointRequest provide
 		return nil, true // skip retry if volume AccessPoint is not found OR already in deleting, pending deletion state
 	})
 	if err != nil {
-		userErr := userError.GetUserError(string(userError.DeleteVolumeAccessPointFailed), err, deleteAccessPointRequest.VolumeID, volumeAccessPoint.ID)
+		userErr := userError.GetUserError(string(userError.DeleteVolumeAccessPointFailed), err, deleteAccessPointRequest.VolumeID, deleteAccessPointRequest.AccessPointID)
```

Comment:
how this was working previously

Reply:
There is no issue only thing is I tried with zero retries as it takes time to execute the case so it was not going inside the loop and this value was not populated , so safer side use the ID from the request. This is just logging and error formatting